code/src/java/pcgen/cdom/content/factset/FactSetDefinitionEnforcer.java

```diff
@@ -79,7 +72,7 @@ public boolean process(LoadContext context, T obj)
 		 * Note, even if the Indirects in list are empty this should pass,
 		 * because they TRIED, right?
 		 */
-		if ((list != null) && !list.isEmpty())
+		if (list != null && !list.isEmpty())
```

Comment:
I fix extraneous parens to be confusing since I have to work out why they were there. Reverted though.